openssl-fips/plan.sh

```diff
@@ -0,0 +1,40 @@
+pkg_name=openssl-fips
```

Comment:
Should this be called `openssl-fips-module` or `...-fips-container` or something? `openssl-fips` to me implies openssl 'the software' itself.

(Not going to block this to bikeshed on this, but what are y'alls thoughts?)

Reply:
We've changed course on this a few times while trying to find the best solution.   It seems most logical to me to align with what we're already successfully doing in Omnibus, and some smart people worked hard to get that right 😉 https://github.com/chef/omnibus-software/commit/7549f2e0b34f712d8958ed73ba76e1c5e77a02a2#diff-df572dc2a02333739f4d3e4da5f7c7c3

Reply:
Yeah, you can never take back the code you commit! I'll defer if y'all have already bikeshedded on it :)